### PR TITLE
docs: copy-edit README

### PR DESCRIPTION
**Pull request by jamesarosen**
This commit improves the readability of the README without adding or changing any of the meaning.

### CODE REVIEW

**Comment by jamesarosen** [comment]
Thank you for the repo, @bxb100!

リポジトリを ありがとう 😄

File: README.md [hunk 1]
```diff
@@ -20,31 +20,31 @@ A Github Action for uploading files to a webdav server
...
+| `webdav_password`         | WebDAV password                                                                                                                                                                       | -       |
+| `webdav_upload_path`      | The WebDAV path where you want to upload. Some servers may not support the root path                                                                                                  | -       |
+| [`files`](#files)         | Newline-delimited list of path globs for asset files to upload <br> :feet: You can learn more about multi-line YAML syntax [here](https://yaml-multiline.info/)                       | -       |
+| `fail_on_unmatched_files` | Fail the action if there exists an unmatched file pattern                                                                                                                             | false   |
+| `keep_structure`          | Keep the directory structure of the files<br/> **Only supports a single search path**, [rule](https://github.com/actions/toolkit/blob/main/packages/glob/src/internal-globber.ts#L27) | false   |
```
Comment (jamesarosen) [inline README.md:36]: I didn't quite understand this line. If you can explain further (in English or Japanese), I'll gladly improve it further.

Comment (bxb100) [reply README.md:36]: Initially, I want to upload some files under the specified directory. but in #51 somebody wants to keep struct like: 

> Local: `dist` =>  WebDAV: `public` (for deployment purposes I think)

I don't want to change a lot, so if set `keep_structure` the `files` only support one path

the code:

https://github.com/bxb100/action-upload-webdav/blob/9e4f49f597ccd11be3791374616bd67849634830/src/main.ts#L32-L37

https://github.com/bxb100/action-upload-webdav/blob/9e4f49f597ccd11be3791374616bd67849634830/src/main.ts#L54-L62

**Comment by bxb100** [comment]
Thanks~

**Pull request merged**